tests/test_ecs.py

```diff
@@ -160,13 +180,16 @@ def test_task_definition_resource(self):
             session_factory=session_factory,
         )
         resources = p.run()
-        self.assertEqual(len(resources), 3)
+        self.assertEqual(len(resources), 2)
         images = set()
         for r in resources:
             for c in r["containerDefinitions"]:
                 images.add(c["image"])
         self.assertEqual(
-            sorted(images), ["nginx:latest", "postgres:latest", "redis:latest"]
+            sorted(images), [
```

Comment:
as noted previously all of these change are extraneous to the feature being added. the right way to handle augment changes for existing tests, is to simply add in a copy of the new recorded api call to each existing test, not to re-record every existing test. a significant portion of the churn in this pr, is because this is not being handled correctly.

Reply:
yes I see this was the incorrect way to do this. apologies

Reply:
i went ahead and fixed this...

Reply:
in this case it was actually even easier to just use a get('tags', []) since it wasn't an additional api call per se.